web/server/codechecker_server/server.py

```diff
@@ -481,6 +481,9 @@ def do_POST(self):
             LOG.warning(str(exn))
             import traceback
             traceback.print_exc()
+
+            itrans = TTransport.TMemoryBuffer(itrans.cstringio_buf.getvalue())
```

Comment:
Please check if there is data in `itrans.cstringio_buf.getvalue()` it is possible that the exception is thrown before the earlier processing of the thrift data.